docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Clustering/KMeans.cs

```diff
@@ -10,38 +10,49 @@ public static class KMeans
     {
         public static void Example()
         {
-            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
-            // as a catalog of available operations and as the source of randomness.
-            // Setting the seed to a fixed number in this example to make outputs deterministic.
+            // Create a new context for ML.NET operations. It can be used for
+            // exception tracking and logging, as a catalog of available operations
+            // and as the source of randomness. Setting the seed to a fixed number
+            // in this example to make outputs deterministic.
             var mlContext = new MLContext(seed: 0);
 
-            // Convert the list of data points to an IDataView object, which is consumable by ML.NET API.
+            // Convert the list of data points to an IDataView object, which is
+            // consumable by ML.NET API.
             var dataPoints = GenerateRandomDataPoints(1000, 123);
 
-            // Convert the list of data points to an IDataView object, which is consumable by ML.NET API.
+            // Convert the list of data points to an IDataView object, which is
+            // consumable by ML.NET API.
             var trainingData = mlContext.Data.LoadFromEnumerable(dataPoints);
```

Comment:
>var [](start = 12, length = 3)

seems like you and Zeeshan are doing different things with var. He's changing some to concrete type. Could you please come up with what 'var' is being changed?

you can write up a small template, like the ones I've created for the samples & xml documentation. Like this:

1) Limit all lines to max 85 chars
2) don't change var OR change var for case A,B,C and not in case E,F